### DISCUSSION TITLE
Clarification on how to use the exclude flags

### DISCUSSION THREAD

**Discussion by LargeTalons** [cross-seed/cross-seed#233] [Q&A] [answered]
Hey guys,

I am trying to figure out how to allow the script to run daily but only allow it to look at recent torrents and not torrents it has already look at in previous runs. 

Theres the "-r, --exclude-recent-search <cutoff>" that I am trying to understand how to use.

If I run the script say, every 6 hours. How exactly do I use this argument to only look at torrents that were added to the client AFTER the last time the script ran?

Is this the right arguement or should I be using "-x, --exclude-older <cutoff>  "? Either way, Im just trying to understand how these work.

Hopefully this make sense. Thank you for reading ;)

**Comment by mmgoodnow** [comment, marked as answer]
> If I run the script say, every 6 hours. How exactly do I use this argument to only look at torrents that were added to the client AFTER the last time the script ran?

`--exclude-older 360`

The -r flag is for selecting old torrents you haven't searched in a while. The -x flag is for selecting new torrents.

**Comment by LargeTalons** [comment]
Hmm,

so If I only wanted to look at new torrents added in the last 6 hours, I
would use "--offset 360" instead?

I appreciate you taking the time to clarify these things for me. I love
this script and I want to learn how I can utilize it more efficiently

**Reply by mmgoodnow** [reply]
Oh my bad, no not offset. `-x` or `--exclude-older`

**Comment by LargeTalons** [comment]
Thank you for helping me get this working!!! Its working exactly as I
wanted now :)

Is it possible to drop the torrents into specific output folders based on
what type they are (Movie, episode, etc)?

**Reply by Jerrk** [reply]
You can use the "outputDir" in your call to specify the location to save your torrent files.

for an example of my usecase:
I use Deluge and i use the "Execute" plugin to run at the completion of torrents which runs this script:
```
#!/bin/bash
torrentid=$1
torrentname=$2
torrentpath=$3

if [[ $torrentpath == "/mnt/user/Media/Seeding/Radarr" ]] ; then
    curl -XPOST http://192.168.1.123:2468/api/webhook \
		--data-urlencode "infoHash=$torrentid" \
		--data-urlencode "name=$torrentname" \
		--data-urlencode "outputDir=/Radarr"
elif  [[ $torrentpath == "/mnt/user/Media/Seeding/Sonarr" ]] ; then
    curl -XPOST http://192.168.1.123:2468/api/webhook \
		--data-urlencode "infoHash=$torrentid" \
		--data-urlencode "name=$torrentname" \
		--data-urlencode "outputDir=/Sonarr"
else
    :
fi
```

So if a completed torrent folder is within the radarr directory, the script will send "outputDir=/Radarr" to cross-seed so the torrent file will get caught in the watch-folder "/Radarr". Deluge then automatically adds torrents from the "/Radarr" directory with the download folder "/mnt/user/Media/Seeding/Radarr" to the "cross-seed" label in my deluge client

hope that helps

**Reply by mmgoodnow** [reply]
that's gonna go away at some point because it doesn't work for rss and periodic searches. I don't exactly know what will replace it yet, but I'll definitely replace it with something.
